src/AltinnCore/Designer/Controllers/RepositoryController.cs

```diff
@@ -60,10 +60,10 @@ public List<Repository> Search(RepositorySearch repositorySearch)
         }
 
         /// <summary>
-        /// Returns a given service repository
+        /// Returns a given app repository
         /// </summary>
         /// <param name="owner">The service owner</param>
-        /// <param name="repository">The service repository</param>
+        /// <param name="repository">The app repository</param>
         /// <returns>The given service repository</returns>
```

Comment:
service

Reply:
Do not follow this one either.. The "repository" variable has the same value as an "app" variable. We should consider renaming the "repository" variables to "app" while we are at it since it contains the same values.

Reply:
As agreed, we keep the repository variables as is.